scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/task/executable/JavaExecutable.java

```diff
@@ -61,6 +62,8 @@
 
     private Map<String, String> metadata;
 
+    private Map<String, Serializable> JobMap;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "JobMap" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)